penpost/penpost-web
# Gaj/fix/yml

#### Purpose:
 > Check all that apply.

### What does this PR do?
 > Clearly and concisely describe changes made. 

Fix CircleCI yml file. 

### Problems Encountered & Resulting Solutions  
> State any problems encountered 

Needed to reconfigure the yml file without the use the orb due to processes not working with a third-party orb.


> Describe your code changes in detail for reviewers.

NA

### Where should the reviewer start?
> What file(s) and line(s) are where changes are made?

Please see `config.yml`, `package.json` & `Postcard-form.js`

### How should this be tested?
> What steps need to be taken to confirm functionality? 

Confirm in CircleCI works inside of the app. You should see no warnings in the console. 

### Steps To Take
> Any questions and possible steps moving forward.

Once tests are added, need to watch build to confirm test files pass.
